core/src/main/java/org/apache/accumulo/core/spi/scan/ScanPrioritizer.java

```diff
@@ -26,6 +26,7 @@
  * configuring, find the documentation for the {@code tserver.scan.executors.} property.
  *
  * @since 2.0.0
+ * @see org.apache.accumulo.core.spi
```

Comment:
Is this proper syntax for the `@see` tag? (i.e. Does this generate a link to the package-level javadoc?)

Reply:
yeah it does, its kinda neat... I have never used this feature before.